### DISCUSSION TITLE
Is it possible to search on a list box?

### DISCUSSION THREAD

**Discussion by arozbiz** [espanso/espanso#1743] [Feature requests and Ideas]
When using the [choice extension](https://espanso.org/docs/matches/extensions/#choice-extension) it's possible to search across the matches (with keyboard input). Is something similar possible with [list boxes](https://espanso.org/docs/matches/forms/#list-box)?

**Comment by smeech** [comment]


**Reply by arozbiz** [reply]
Whoops! Thanks for the catch. Will edit my question.

**Reply by smeech** [reply]
Thanks - I'll delete my comment.
